packages/components/pie-modal/src/modal.scss

```diff
@@ -65,26 +65,36 @@
     }
 
     & .c-modal-header {
-        --modal-header-padding: var(--dt-spacing-e);
+        --modal-header-padding: var(--dt-spacing-c);
+        padding-inline: var(--dt-spacing-d);
+        padding-block: 14px;
+        @media (min-width: $breakpoint-wide) {
```

Comment:
Can you put the `@media` block after the other declarations in this style block?